app/services/event_processor/delete_processor.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class EventProcessor
+  class DeleteProcessor < Processor
+    def process
+      index.delete_by_query "id:#{RSolr.solr_escape(id)}"
+      index.commit unless bulk?
```

Comment:
Just to be sure, you tested this on staging and it was fine?

Reply:
Yes. It worked. We should setup auto-commits on the orangelight solr cores, though. For bulk updates.